package/rdspgbadger.py

```diff
@@ -151,11 +153,12 @@ def main():
         logger.info("File(s) downloaded. Not processing with PG Badger.")
     else:
         logger.info("Generating PG Badger report.")
-        command = ("{} -p \"%t:%r:%u@%d:[%p]:\" -o {}/report.html "
-                   "{}/error/*.log.*".format(pgbadger, args.output,
-                                             args.output))
-        logger.debug("Command:")
-        logger.debug(command)
+        command = ("{} -p \"%t:%r:%u@%d:[%p]:\" {} -o {}/report.html"
```

Comment:
The extra space was intended to keep this parameter separated from the next one.

Reply:
Ah, my bad - I'll fix.  I'll also handle updating the README.